frontend/src/components/common/button/button.tsx

```diff
@@ -36,7 +41,11 @@ const Button: FC<Props> = ({
   return (
     <button
       type={type}
-      className={getValidClasses(styles.button, styles[`button-${btnColor}`])}
+      className={getValidClasses(
+        styles.button,
+        styles[`button-${btnColor}`],
+        btnType === 'outlined' && styles.outlined,
```

Comment:
Let's use same approach to pass `outlined | filled` type as it's done with `btnColor` passing

Reply:
agreed